### ISSUE TITLE
Bulk create for performance

### ISSUE THREAD

**Issue by srtab** [pinax/django-mailer#103]
Hi, 

I have a question related to the application's performance and, perhaps, an improvement.
In the `send_messages` method, in https://github.com/pinax/django-mailer/blob/master/mailer/backend.py#L8, why not using `bulk_create` https://docs.djangoproject.com/en/2.1/ref/models/querysets/#bulk-create, instead of saving one by one? This will significantly increase performance when sending a big amount of emails at once.

Tks and congrats for the great application :)
Sandro Rodrigues

**Comment by spookylukey** [comment]
@srtab  Most likely we are not using `bulk_create` because that API didn't exist when that code was written. Patch gladly accepted!

**Comment by srtab** [comment]
Ok @spookylukey, maybe next week i make the pull request with proposed changes. Tks.

**Comment by srtab** [comment]
Hi @spookylukey,
Just a question, hhen is the next release scheduled? 
Tks

**Comment by srtab** [comment]
Yes @spookylukey, but for production environment's is not a good / recommendable choice to install a dependency.

**Comment by spookylukey** [comment]
@srtab - why is that?
